### DISCUSSION TITLE
FGD212 not working/regonized

### DISCUSSION THREAD

**Discussion by Frank18000** [zwave-js/zwave-js#6697] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
The dimmer is regonized. But know configuration vissable and know action when the dimmer is activate via Home assistent. Lights are working through te wall switch.

What did you expect to happen instead?

Full control over the node.

  2. Click on '...'
  3. Scroll down to '...'

Anything else we should know?

### Software versions

Driver (node-zwave-js): 3.4.1

Z-Wave JS UI: ...

Home Assistant Z-Wave Integration: 1.33.0

Home Assistant Z-Wave JS Addon: ja

ioBroker.zwave2 Adapter: 700 series 7.17.2

If you are using something non-standard, tell us here: ...

### Device information

Manufacturer: Fibaro
Model name: FGD212
Node ID: 0x0102

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwave-js-ui-store (1).zip](https://github.com/zwave-js/node-zwave-js/files/14601988/zwave-js-ui-store.1.zip)
[home-assistant_2024-03-14T12-07-47.244Z.log](https://github.com/zwave-js/node-zwave-js/files/14601998/home-assistant_2024-03-14T12-07-47.244Z.log)

**Comment by AlCalzone** [comment]
> The log includes a re-interview of the problematic device

It doesn't. Please re-interview the device and share a log of that.
